packages/terra-embedded-content-consumer/src/terra-dev-site/test/embedded-content-consumer/consumers/Consumer.module.scss

```diff
@@ -0,0 +1,6 @@
+:local {
```

Comment:
we need to add these styles because before we were getting them through the weirdness of not providing the forward slash.